Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -19,25 +20,29 @@ class TransactionHistoryViewController: UIViewController {
         return refreshControl
     }()
     
-    @IBOutlet weak var syncLabel: UILabel!
     
+    @IBOutlet weak var syncLabel: UILabel!
     @IBOutlet var tableView: UITableView!
     @IBOutlet var btnFilter: DropMenuButton!
-    var FromMenu = true
-    var visible:Bool = false
     
-    var filterMenu = [LocalizedStrings.all] as [String]
-    var filtertitle = [0] as [Int]
+    var filterOptions = [LocalizedStrings.all]
```

Comment:
This is only used in the `initFilterBtn` method and so does not need to be global. Also check if it's possible to do without it altogether. Is it possible to init the filter dropdown using an empty array? The correct array of filter options is set later in the `updateFilterDropdownItems` method; anything set here/now is pretty much just useless.

Reply:
fixed in 28f46d1